src/typography/index.md

```diff
@@ -25,15 +25,17 @@ The Government Digital Service adapted it in 2012 for digital use, bringing the
 
 Using GDS Transport makes GOV.UK easier to recognise and information easier to read on any device.
 
-![TODO](./transport-font.svg)
+<div class="scale width-80 right edge">
+
+![Sample of GDS Transport font in bold and regular weight](./transport-font.svg)
```

Comment:
You hadn't addressed the other of my previous comments:

> It's one typeface but two fonts. So, technically this sentence is incorrect. (If I understood that correctly?)
> Same for the other one occurrence below.

That means I would either change this to:

> Sample of GDS Transport typeface in bold and regular weight

Or to:

> Sample of GDS Transport bold font and regular font

Reply:
And as per my recent comment, where you say "regular" it needs to say "light".

So either:

> Sample of GDS Transport typeface in bold and light weight

Or:

> Sample of GDS Transport bold font and light font